### PR TITLE
Base-theme

### PR DESCRIPTION
**Pull request by btzr-io**
### Todo

- [x] Remove old theme.

- [x] Update new themes.

- [x] Implement installer.

- [x] Test.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <html>
   <head>
       <title>Butter Component Tester</title>
+      <link  rel="stylesheet" type="text/css" href="./node_modules/butter-base-theme/index.css"/>
```
Comment (xaiki) [inline index.html]: i guess it should be named `butter-theme-default` or else it's going to be hard to make an installer

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "babel-preset-stage-0": "^6.0.15",
     "bootstrap-loader": "^2.0.0-beta.21",
     "bootstrap-sass": "^3.3.7",
+    "butter-base-theme": "^0.1.2",
```
Comment (xaiki) [inline package.json]: ditto

File: styl/fonts.styl [hunk 3]
```diff
@@ -1,55 +0,0 @@
...
-/*    Material Icons    */
-
-@font-face
-    font-family: 'Material Icons'
-    font-style: normal
-    font-weight: 400
-    src: local('Material Icons'),
-         local('MaterialIcons-Regular'),
-         url(../fonts/Material-icons/MaterialIcons-Regular.woff2) format('woff2'),
-         url(../fonts/Material-icons/MaterialIcons-Regular.woff) format('woff')
```
Comment (xaiki) [inline styl/fonts.styl:44]: this is required by material icon, it's not a theming thing.

Comment (btzr-io) [reply styl/fonts.styl:44]: WTF? -> https://github.com/buttercomponents/butter-component-builder/blob/master/styl/app.styl#L13

File: index.html (hunk 1 above)
Comment (btzr-io) [reply index.html]: installer??

File: package.json (hunk 2 above)
Comment (btzr-io) [reply package.json]: Don't see nothing wrong with the name...

File: index.html (hunk 1 above)
Comment (btzr-io) [reply index.html]: It's not a theme or default-theme is a `base` to build themes!

Comment (btzr-io) [reply index.html]: we import it to `app.styl` or inside `index.html`

Comment (btzr-io) [reply index.html]: Not sure why we need an installer? ( care to explain? )
it's on npmjs.com so: `$ npm i butter-base-theme`

File: styl/fonts.styl (hunk 3 above)
Comment (xaiki) [reply styl/fonts.styl:44]: i am perplexed as to how dat shit works

Comment (btzr-io) [reply styl/fonts.styl:44]: I think we use a module, I'll check...

Comment (btzr-io) [reply styl/fonts.styl:44]: https://github.com/buttercomponents/butter-component-builder/blob/master/webpack.config.js#L10

**Comment by btzr-io** [comment]
https://github.com/butterthemes/butter-base-theme/issues/1

File: package.json (hunk 2 above)
Comment (xaiki) [reply package.json]: our naming scheme is `butter-$thing-$name`; `butter-component-settings`, `butter-streamer-http`, `butter-theme-blah`

**Comment by btzr-io** [comment]
@xaiki I think we should merge / implement this first

**Comment by btzr-io** [comment]
@xaiki I need this so I can start fixing `styles + layout` ^^

**Comment by btzr-io** [comment]
I'm going to merge this!

File: package.json [hunk 4]
```diff
@@ -37,9 +37,7 @@
     "babel-preset-stage-0": "^6.0.15",
     "bootstrap-loader": "^2.0.0-beta.21",
     "bootstrap-sass": "^3.3.7",
-    "butter-base-theme": "^0.1.2",
-    "butter-theme-dark": "^0.1.3",
-    "butter-theme-pink": "^0.0.1",
+    "butter-installer-theme": "^0.1.0",
```
Comment (xaiki) [inline package.json]: this should be just butter-base-theme (that should be called butter-theme-default or butter-theme-base)

**Review by xaiki — COMMENTED** [review]
not sure this is ok

File: package.json (hunk 4 above)
Comment (btzr-io) [reply package.json]: > (actually it should be butter-installer-themes, as to not confuse it with a theme package)

Comment (xaiki) [reply package.json]: but ... it's not a theme installer...

Comment (btzr-io) [reply package.json]: Well is not a theme either, not sure what you mean by that, but feel free to send a PR to make it an `"installer"`:

https://github.com/butterthemes/butter-installer-themes

Comment (btzr-io) [reply package.json]: Well if you `npm install` this, it should install,
Base-theme + dark theme + ...
That's looks like an installer to me ^^

Comment (btzr-io) [inline package.json]: Well if you `npm install this, it should install,
Base-theme + dark theme + ...
That's looks like an installer to me ^^

Comment (xaiki) [reply package.json]: it's a pack, anyway as we discussed on slack please remove it and load themes that are in package.json

Comment (btzr-io) [reply package.json]: #9 ^^

**Pull request merged**